Cargo.toml

```diff
@@ -122,6 +125,9 @@ libp2p-tls = { version = "0.1.0-alpha", path = "transports/tls", optional = true
 [target.'cfg(not(target_os = "unknown"))'.dependencies]
 libp2p-gossipsub = { version = "0.43.0", path = "protocols/gossipsub", optional = true }
 
+[patch.crates-io]
+webrtc = { version = "0.5.0", git = "https://github.com/webrtc-rs/webrtc.git", branch = "anton/poll-stream-fixes-2" }
```

Comment:
For the record, merge is blocked on this.

Reply:
This is now fixed 👍